src/core/StelVertexArray.cpp

```diff
@@ -55,14 +56,13 @@ StelVertexArray StelVertexArray::removeDiscontinuousTriangles(const StelProjecto
 	else
 	{
 		ret.indices.clear();
-		unsigned short int limit;
+		const unsigned short int limit=static_cast<unsigned short int>(vertex.size());
```

Comment:
It may be a good idea to assert that `vertex.size() <= USHRT_MAX` (or `std::numeric_limits<unsigned short>::max()`).

Also, this line could be shortened by removing repetition from the LHS and the useless ` int` from the RHS. Or just using an implicit cast.